docs/src/ug/routerules.md

```diff
@@ -62,15 +62,15 @@ They share the same schema but each Input Variable controls the associated route
 - `network_entity_id` accept string value and represent the gateway to be the target of the rule,
 - `description` accept arbitrary string value and give context about the goal of the rule.
 
-#### special values for `network_entity_id`
+#### Special values for `network_entity_id`
 
 For routing rules targeting a Gateway created by the module, `network_entity_id` accepts some special strings to automatically retrieve the gateway ocid.
 
 - `internet_gateway_route_rules` and `nat_gateway_route_rules` recognise the `"drg"` string and resolve it to the DRG ocid attached to the module (if available),
```

Comment:
If DRG isn't defined earlier, it needs to be expanded out here. Also, OCID is an acronym so it needs to be in capitals.